app/models/package_manager/go.rb

```diff
@@ -21,32 +23,71 @@ class Go < Base
 
 
     def self.package_link(project, version = nil)
-      "http://go-search.org/view?id=#{project.name}"
+      "https://pkg.go.dev/#{project.name}#{"@#{version}" if version}"
     end
 
     def self.documentation_url(name, version = nil)
-      "http://godoc.org/#{name}"
+      "https://pkg.go.dev/#{name}#{"@#{version}" if version}?tab=doc"
     end
 
     def self.install_instructions(project, version = nil)
       "go get #{project.name}"
     end
 
     def self.project_names
```

Comment:
I wonder if it's worth jumping on https://github.com/golang/go/issues/36785 and joining that discussion? We should at least be following it

Reply:
Good idea, hadn't seen that yet! Just posted some thoughts: https://github.com/golang/go/issues/36785#issuecomment-639794685